src/java/org/apache/nutch/scoring/ScoringFilter.java

```diff
@@ -179,6 +179,20 @@ public void updateDbScore(Text url, CrawlDatum old, CrawlDatum datum,
       List<CrawlDatum> inlinked) throws ScoringFilterException;
 
   /**
+   * This method may change the score or status of CrawlDatum during CrawlDb
+   * update, when the URL is neither fetched nor has any inlinks.
+   *
+   * @param url
+   *          URL of the page
+   * @param datum
+   *          CrawlDatum for page
+   * @throws ScoringFilterException
```

Comment:
I think this 'may' break Javadoc generation if no comment is provided alongside the Exception itself.